tests-functional/conftest.py

```diff
@@ -19,14 +19,10 @@ def pytest_addoption(parser):
         default="ws://0.0.0.0:8354",
     )
     parser.addoption(
-        "--status_backend_urls",
+        "--status_backend_url",
         action="store",
         help="",
-        default=[
-            f"http://0.0.0.0:{3314 + i}" for i in range(
-                int(os.getenv("STATUS_BACKEND_COUNT", 10))
-            )
-        ],
+        default="http://0.0.0.0:3334",
```

Comment:
I think we should leave the default value empty now

Reply:
agree, and reverse condition for url so we can run status-backend with binary